src/Modal/styles.css

```
@@ -0,0 +1,35 @@
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: rgb(84, 64, 187);
+  border: solid rgb(200, 144, 21) 0;
+  outline: solid black 0.1em;
+  border-radius: 0.5rem;
+  padding: 2em;
+  z-index: 1000;
+  visibility: hidden;
+  width: 80%;
+  max-width: 450px;
```

```suggestion
  max-width: 30rem;
```
Prefer using rem / em values to save time later when mobile mode gets fixed lol